src/Diagnostics.ModelsAndUtils/ScriptUtilities/SolutionTypeTag.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.ScriptUtilities
+{
+    public enum SolutionTypeTag
```

Comment:
>SolutionTypeTag [](start = 16, length = 15)

SolutionType? More readable...

Reply:
This is just to match `enum SolutionTypeTag` from the Angular project